src/config/cli.rs

```diff
@@ -139,6 +143,10 @@ pub struct Cli {
     /// Log file path (default: era_test_node.log).
     pub log_file_path: Option<String>,
 
+    #[arg(long, default_missing_value = "true", num_args(0..=1), help_heading = "Logging Configuration")]
+    /// If true, the tool will not print anything on startup.
+    pub silent: Option<bool>,
```

Comment:
This is now `quiet` in anvil:

```
  -q, --quiet
          Do not print log messages
```

Reply:
wdym? I don't see `--quiet` in anvil, it's `--silent`